web/admin/src/pages/products/add.vue

```diff
@@ -126,11 +146,11 @@ const handleFormSubmit = async () => {
   }
 }
 
-const getErrorMessage = (key: string): string | Ref<string> => {
+const getErrorMessage = (key: string): string => {
   const error = v$.value.$errors.find((e) => {
     return e.$property === key
   })
-  return error ? error.$message : ''
+  return error ? `${error.$message}` : ''
```

Comment:
messageがundefinedの時を考慮すると↓の方が良かったりする?

```suggestion
  return error?.$message || ''
```

Reply:
あ、ここはmessageがundefinedの可能性はなくて、messageの型は`string | globalThis.Ref<string>`なのよね👀

なので文字列展開すれば、エラーメッセージは取れると思っています🙏